toonz/sources/include/toonzqt/paletteviewergui.h

```diff
@@ -116,6 +116,9 @@ class DVAPI PageViewer final : public QFrame, public TSelection::View {
   NameDisplayMode getNameDisplayMode() const { return m_nameDisplayMode; }
   void setNameDisplayMode(NameDisplayMode mode);
 
+  void toggleShowStyleIndex();
```

Comment:
I'm leaving comments for Ryan:
This makes the style indexes in the corner toggle-able.  I'm not sure that is desired for OpenToonz.